intelmq/bots/parsers/bambenek/parser.py

```diff
@@ -32,32 +33,33 @@ def parse_line(self, line, report):
             self.tempdata.append(line)
 
         else:
-            value = line.split(',')
+            m = re.match(r"(?P<ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}),(?P<description>.*), \
+                (?P<timestamp>\d{4}-\d{2}-\d{2}[ ]\d{2}[:]\d{2}),(?P<url>.*)", line)
```

Comment:
Could we change `.*` (greedy) to `.*?` (non-greedy) here?

Can we `$` at the end to indicate the regular expression should match the full line?